### PR TITLE
VP Role Expansion and Addition of Marketing Officer

### PR DESCRIPTION
**Pull request by innocentmm**
See comimt details

### CODE REVIEW

File: Constitution.md [hunk 1]
```diff
@@ -317,4 +315,4 @@ Prepare necessary forms, and receipts for Societies Council in accordance with S
 
 31.1 If upon the dissolution of the club, any property of the club remains after satisfaction of the debts and liabilities of the club that property shall be distributed to another association or club affiliated to the Societies Council which has similar objectives to those of the club, and which association shall be determined by resolution of the members.
 
-Adopted by General Meeting ______________________________________ 20/03/2018
\ No newline at end of file
+Adopted by General Meeting **\*\*\*\***\*\***\*\*\*\***\_\_**\*\*\*\***\*\***\*\*\*\*** xx/xx/2020
```
Comment (Drewbi) [inline Constitution.md]: What's going on here?

File: Constitution.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ Students enrolled at the University of Western Australia.
 
 **5. SUBSCRIPTIONS**
 
-5.1 The annual subscription for ordinary membership shall be an absolute minimum of $4.
+5.1 The annual subscription for ordinary membership shall be an absolute minimum of \$4.
```
Comment (Drewbi) [inline Constitution.md]: We can do another PR for formatting

**Comment by innocentmm** [comment]
Oh whoops, we haven’t had the AGM so I didn’t set a date but I don’t know what’s happening there otherwise

Kind regards,

Innocent.
________________________________

**Review by Drewbi — CHANGES_REQUESTED** [review]
This doesn't really transfer the responsibilities of Client Officer onto VP it just replaces it with Marketing Officer.

**Comment by innocentmm** [comment]
Changes have been made.....

**Review by Drewbi — APPROVED** [review]

**Review by Drewbi — APPROVED** [review]

**Pull request closed without merging**